### PR TITLE
[RFC][UN-8793] Create uniq 2.0 components   calendar

### PR DESCRIPTION
**Pull request by mmelo**
Refers to: https://uniplaces.atlassian.net/browse/UN-8793

fixing calendar styles
screen shot 2017-05-02 at 14 19 40
screen shot 2017-05-02 at 14 20 08
screen shot 2017-05-02 at 14 20 21

### CODE REVIEW

File: addon/styles/partials/_uni-datepicker.scss [hunk 1]
```diff
@@ -2,31 +2,68 @@ $box-size: 42px;
 
 .ember-power-calendar {
   width: 100%;
-  max-width: 294px;
+  max-width: 300px;
+  padding: 20px 10px 10px;
+  border: 1px solid color('primary', 'divider');
```
Comment (AndreJoaquim) [inline addon/styles/partials/_uni-datepicker.scss]: Should this be primary divider or should we create a new calendar palette? 🤔

File: addon/styles/partials/_uni-datepicker.scss [hunk 2]
```diff
@@ -44,7 +81,7 @@ $box-size: 42px;
     background-color: color('buttons') !important;
 
     &:hover {
-      background-color: color('buttons', 'hover') !important;
+      background-color: color('dropdown', 'active') !important;
```
Comment (AndreJoaquim) [inline addon/styles/partials/_uni-datepicker.scss]: We shouldn't have dropdown colors on calendar

File: addon/styles/partials/_uni-datepicker.scss [hunk 3]
```diff
@@ -53,18 +90,19 @@ $box-size: 42px;
...
 
   &-day--other-month {
-    background-color: color('buttons', 'disabled');
+    background-color: color('buttons', 'disabled') !important;
+    color: color('background', 'divider');
   }
 
   &-day--selected {
-    color: color('primary') !important;
+    background-color: color('dropdown', 'active') !important;
```
Comment (AndreJoaquim) [inline addon/styles/partials/_uni-datepicker.scss]: Same here

File: addon/styles/partials/_uni-datepicker.scss [hunk 4]
```diff
@@ -53,18 +90,19 @@ $box-size: 42px;
...
   }
 
   &-day--interactive[disabled] {
     opacity: 1 !important;
     text-decoration: none;
     background-color: color('buttons', 'disabled') !important;
-    color: rgba(178, 185, 194, .3);
+    color: color('dropdown', 'divider');
```
Comment (AndreJoaquim) [inline addon/styles/partials/_uni-datepicker.scss]: Same here

**Review by AndreJoaquim — CHANGES_REQUESTED** [review]

**Review by montoias — APPROVED** [review]

**Review by AndreJoaquim — APPROVED** [review]

**Pull request merged**